data/human/human missions.txt

```diff
@@ -6438,6 +6438,407 @@ mission "Care Package to South 2"
+			`		Sincerely,`
+			`			Artemis Renard`
+				accept
+	on complete
+		log "Was given a Racing Flivver as additional compensation for helping find Artemis Renard."
+		conversation
+			`As Glory again rises to greet your ship, you receive a hail from a spacecraft that comes up alongside your ship. "Ho, Captain! Why don't you come to our estate, instead of the main landing port? It'll be easier." August Renard is leaning out the cockpit of a fancy Racing Flivver, looking much happier than when you first met him.`
+			`	You redirect your ship in the direction the Flivver is going, noticing that it has the same name as Artemis' ship which crashed on Sandswell.`
+			`	Soon, the dense main city of <planet> gives way to a perfectly-terraformed countryside with only one mar - a gigantic mansion that must be hundreds of meters high. A few landing pads are set in a neat array next to it, and the B-168 "Moonfox" lands on one of these, beckoning for you to land on another.`
+			choice
+				`	(Land on one of the pads.)`
+					goto continue
+				`	"What about my fleet?"`
+					to display
+						"total ships" > 1
+					goto continue
+				`	"You have a mansion here!"`
+			`	"Later, later," Artemis shouts. "We'll talk in a place where we needn't shout."`
+			label continue
+			`	August calls, "You can let your fleet stay at the main spaceport; I'm afraid there's not room here."`
+				to display
+					"total ships" > 1
+			`	As you exit <ship>, you notice that the Flivver has been painted in the same colors that your ship has.`
+			`	August approaches. "Artemis wanted to give you something as a present for saving her life, and we were wondering if you would like to have this Flivver. We've managed to refurbish it enough that it should be perfectly functional, and if you want a fast ship there's nothing in the sky that can beat you."`
+			`	August leads you up the steps into the palatial mansion and into a large whitewashed room with a golden table. On the table lie keys for the starship Artemis used in her most recent race.`
+			action
+				give ship "Racing Flivver" `B-168 "Moonfox"`
+			`	"Well, I hope you like it." Artemis approaches. "It's been servicing me for quite a while, ever since I started racing. Unfortunately it's not fast enough anymore, even after the millions we spent on rebuilding it after the crash on Sandswell. Hopefully you'll be able to put it to better use." She smiles. "Of course, it would also be a good ship to sell, if you need the money."`
+			`	After spending another half an hour or so catching up, you say your goodbyes to the pair and head back to the main spaceport.`
+
+ship "Racing Flivver"
```

Comment:
Neat ship, and very pretty! Have not done a balance review yet - will probably wait until we get a confirmation from devs on whether adding this as a one off ship is desired or not.